flink-runtime/resources/templates/sorter.ftlh

```diff
@@ -35,6 +35,9 @@ import java.util.List;
 import org.apache.flink.runtime.operators.sort.InMemorySorter;
 import org.apache.flink.runtime.operators.sort.LargeRecordHandler;
 
+/**
+ * This code is generated on ${.now}
```

Comment:
@ggevay do you where can I get Flink's `version? I think it will be useful for debugging if the generated code also contains the version.